docs/guide/snaps-rpc-api.md

```diff
@@ -0,0 +1,579 @@
+# Snaps JSON-RPC API
+
+::: tip Only Available in MetaMask Flask
+Snaps is only available in [MetaMask Flask](https://metamask.io/flask).
+:::
+
+::: tip Feature Requests
+Do you have feature requests? Other ideas? We'd love to hear about them! [Click here](https://community.metamask.io/c/metamask-flask) to join the discussion.
+:::
+
+## Table of Contents
+
+[[toc]]
+
+## Unrestricted Methods
+
+### `wallet_enable`
+
+::: warning Only Callable By
+
+- Websites
```

Comment:
I'm surprised to see these methods here. I was thinking of the "Snaps API" as "the API provided to snaps". The idea being that this would be one cohesive API.  I would expect the snap-related methods offered to websites to be in our main RPC API docs.

Reply:
Right, and currently it's "Snaps API" as in "the API related to snaps, regardless of to whom / what it's exposed". I don't know that the website-only methods map neatly on the existing JSON-RPC API page either, in that snaps can call many / some of those methods as well (although perhaps they shouldn't be able to by default).

Reply:
We can explain the subset of the RPC API that snaps get access to in the Snaps reference documentation. I think that would be the simplest way to organize this. The RPC API docs are somewhat of a mess right now because of the feature parity issue between mobile and extension, but at least aspirationally we want to converge at some point on a shared API that is standardized between all products. Since dapps will rely on these snap-related methods, they belong there.

Reply:
Is this something that needs to be resolved before tomorrow morning?

Reply:
It's important that we resolve it soon, for the sake of starting snaps off with a solid foundation in terms of the structure of the documentation. But I don't think it's strictly needed for tomorrow morning, no